packages/internal/dex/sdk/package.json

```diff
@@ -11,6 +11,7 @@
     "ethers": "^5.7.2"
   },
   "devDependencies": {
+    "@imtbl/config": "0.0.0",
```

Comment:
This was added to `dependencies` in an unrelated PR https://github.com/immutable/ts-immutable-sdk/pull/1934